content/intro/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+linkTitle: अवलोकन
+title: Brigade की अवलोकन
+description: Brigade का उच्च स्तरीय दृश्य
+section: परिचय
+weight: 1
+aliases: 
+  - /overview.md
+  - /intro/overview.md
+---
+
+Brigade एक इवेंट संचालित स्क्रिप्टिंग टूल है जो Kubernetes पर चलता है। इसका मतलब यह है:
+
+- Brigade क्लाउड में स्क्रिप्ट योग्य स्वचालित कार्यों को चलाने के लिए है।
+- Brigade के लिए आपको कोई होस्ट सर्वर प्रबंधित करना ज़रुरी नहीं है।
+-Brigade विशेष रूप से सीआई और सीडी वर्कलोड के लिए उपयुक्त है जैसे:
+- Automated testing
+  - GitHub hook integration
```

Comment:
Have these 4 points been intentionally left in English? I would recommend translating them too.

Reply:
Shall I just write the Hindi translations of it in brackets alongside?

Reply:
"Automated testing" can be simply "स्वचालित परीक्षण".
If it were proper nouns or acronyms, we could go with the model that we decided on.

For some not-so-straightforward translations like "artifacts", we can also use the transliteration model.